src/lib/execution.ts

```diff
@@ -42,20 +42,20 @@ import {
  * @param merkleRoot - Optional merkleRoot of the CommitReport, for validation
  * @returns ManualExec report arguments
  **/
-export function calculateManualExecProof(
-  messagesInBatch: readonly CCIPMessage[],
-  lane: Lane,
+export function calculateManualExecProof<V extends CCIPVersion>(
```

Comment:
Let's not require explicitly the template type to be passed
```suggestion
export function calculateManualExecProof<V extends CCIPVersion = CCIPVersion>(
```